### DISCUSSION TITLE
Bimontly smol maintainer's meeting

### DISCUSSION THREAD

**Discussion by notgull** [smol-rs/smol#299] [General]
Hello @smol-rs/admins @pnevyk @Jules-Bertholet,

I am interested in having a general meeting for `smol-rs` maintenance items, such as open pull requests/issues, potential design decisions, and overall maintenance of `smol-rs`. As the maintenance burden of `smol-rs` is generally low, I'm fine with this meeting being only bimonthly except in case of emergencies (major bugs, security vulns, etc).

I'm in PST, what time would work for all of you?

**Comment by pnevyk** [comment]
Hi, I only worked on [fastrand-contrib](https://github.com/smol-rs/fastrand-contrib) crate. I am happy to help with maintaining that crate, or even `fastrand` itself. But when it comes to `smol-rs` itself, my knowledge of async Rust is very basic and – to be honest – I am not interested in async Rust, at least not currently. So I guess it wouldn't make much sense for me to join such meetings, and I would rather keep the communication in asynchronous and textual form. I hope that's ok.

**Comment by fogti** [comment]
I'm in CET / CEST, my available times fluctuate a bit tho (due to university and similar stuff), vary mostly aligned with day-of-week usually.

**Comment by zeenix** [comment]
I'd be good with bimonthly meetings. I'm also in CET/CEST but I usually wake up very late and work till late too. Hence this comment coming in at 11:18 PM. :)

**Comment by notgull** [comment]
@taiki-e Would you be available at any consistent time for this meeting? If not, that's okay.

@zeenix, @fogti How would you feel about Wednesday at 16 UTC?

**Reply by zeenix** [reply]
> @zeenix, @fogti How would you feel about Wednesday at 16 UTC?

Should be good for me.

**Reply by taiki-e** [reply]
> @taiki-e Would you be available at any consistent time for this meeting? If not, that's okay.

Sorry, I don't usually participate in these types of real-time meetings.

**Reply by fogti** [reply]
I should be available I suppose. (Unless public transit is an absolute mess on that day, because I have an exam on that day and usually have no uplink during transit)

**Comment by notgull** [comment]
@fogti @zeenix Sorry for the delay in action on this. I've privately sent you both invitations for a meeting on the 17th at 15:30 UTC. Let me know if this date/time works.
